src/Admin/IdeasWorkshopQuestionAdmin.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use Doctrine\ORM\QueryBuilder;
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\AdminBundle\Show\ShowMapper;
+
+class IdeasWorkshopQuestionAdmin extends AbstractAdmin
+{
+    public function createQuery($context = 'list')
+    {
+        /** @var QueryBuilder $proxyQuery */
+        $proxyQuery = parent::createQuery($context);
+
+        $proxyQuery
+            ->leftJoin('o.guideline', 'g')
+            ->orderBy('g.position', 'ASC')
```

Comment:
It seems that doing so will disable other sort possibilities (by name or category) on the list page.